index.bs

```diff
@@ -302,6 +302,23 @@ and defensive programming which includes:
 
 <h2 id="security-and-privacy">Security and privacy considerations</h2>
 
+<div class="note">
+The judgement on how to communicate to the user the known [[#main-privacy-security-threats|threats]]
+and mandatory [[#mitigation-strategies|mitigations]] used to protect the user against these threats
```

Comment:
I read this sentence few times and it sounds like implementer has to communicate about "threats **and** mitigation strategies". IMO, implementer has to communicate about threats and **implement** mitigation strategies to reduce risks. Information about mitigations are technical and not relevant to the end user.